_clients/OSC-dot-net.md

```diff
@@ -0,0 +1,329 @@
+---
+layout: default
+title: Getting started with OpenSearch.Client
+nav_order: 10
+has_children: false
+parent: .NET clients
+---
+
+# Getting started with OpenSearch.Client
+
+OpenSearch.Client is a high-level .NET client on top of OpenSearch.Net. It provides strongly typed requests and responses, as well as Query DSL. It frees you from constructing raw JSON requests and parsing raw JSON responses by providing models that parse and serialize/deserialize requests and responses automatically. OpenSearch.Client also exposes the OpenSearch.Net low-level client if you need it.
+
+This getting started guide illustrates how to connect to OpenSearch, index documents, and run queries.
+
+## Installing OpenSearch.Client
+
+To install OpenSearch.Client, download the OpenSearch.Client NuGet package and add it to your project in an IDE of your choice. In Microsoft Visual Studio: 
```

Comment:
Should we link off to the location? Is this it? https://www.nuget.org/packages/Terradue.OpenSearch.Client/

Reply:
I will update the location once the package is uploaded to nuget. Thanks!

Reply:
Disregard, discussed offline that it's not on Nuget yet https://github.com/opensearch-project/opensearch-build/issues/2051